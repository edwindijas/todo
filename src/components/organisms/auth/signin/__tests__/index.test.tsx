import Signin from '..'
import render from '@/utils/test-utils/customRender'

describe('Signin Organisms', () => {
  it ('matches snapshot', () => {
    const {container} = render(<Signin />)
    expect(container).toMatchSnapshot()
  })
})