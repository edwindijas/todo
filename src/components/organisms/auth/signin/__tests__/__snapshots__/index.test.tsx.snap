// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signin Organisms matches snapshot 1`] = `
<div>
  <div>
    <h1>
      sign in
    </h1>
    Sign in with your taskspace account
    <div
      class="flex gap-4 flex-col py-4"
    >
      <label
        class="undefined  flex relative cursor-text border-[#DCE2ED] bg-[#F9FBFD] fill-blue-desaturated border rounded-lg h-14 px-4 capitalize"
      >
        <div
          class="shrink-0 grow-0 flex justify-center items-center"
        >
          <div
            class="w-6 h-6"
          >
             
            <svg
              class=" w-full h-full"
              data-testid="component-atom-icon"
            />
          </div>
        </div>
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-desaturated opacity-100"
          >
            email
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-[#DCE2ED] bg-[#F9FBFD] fill-blue-desaturated border rounded-lg h-14 px-4 capitalize"
      >
        <div
          class="shrink-0 grow-0 flex justify-center items-center"
        >
          <div
            class="w-6 h-6"
          >
             
            <svg
              class=" w-full h-full"
              data-testid="component-atom-icon"
            />
          </div>
        </div>
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-desaturated opacity-100"
          >
            password
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
        <div
          class="shrink-0 grow-0 flex justify-center items-center"
        >
          <div
            class="w-6 h-6"
          >
            <svg
              class=" w-full h-full"
              data-testid="component-atom-icon"
            />
          </div>
        </div>
      </label>
      <button
        class="font-medium flex w-full capitalize items-center gap-2 relative text-center text-white bg-blue-medium px-2 h-14 rounded-md"
        data-testid="button-btn"
      >
        <span
          class="w-full"
        >
          continue
        </span>
      </button>
    </div>
    <p>
      need somthing else
    </p>
    <a
      class=" flex w-full capitalize items-center gap-2 relative px-2 h-14 rounded-md"
      href="/"
    >
      <span
        class="w-full"
      >
        create free account
      </span>
    </a>
  </div>
</div>
`;
